### PR TITLE
Update supported versions

### PR DESCRIPTION
**Pull request by fhemberger**
Updated supported versions, as mentioned in #1831.

However, I noticed that the two documents seem to be drifting apart in other sections. So we should really think about how we want to handle this in the future. That's also the reason I didn't update all distros.

/cc @bnb @rvagg Any ideas on how to proceed?

### CODE REVIEW

File: locale/en/download/package-manager.md [hunk 1]
```diff
@@ -78,21 +78,24 @@ sudo apt-get install -y build-essential
 
 **Available architectures:**
 
-* **i386** (32-bit)
+* **i386** (32-bit)—**not available for Node.js 10 and later**
```
Comment (vsemozhetbyt) [inline locale/en/download/package-manager.md]: Nit: let's use m-dash or hyphen with spaces consistently (compare additions below)?

File: locale/en/download/package-manager.md [hunk 2]
```diff
@@ -78,21 +78,24 @@ sudo apt-get install -y build-essential
...
 * **amd64** (64-bit)
 * **armhf** (ARM 32-bit hard-float, ARMv7 and up: _arm-linux-gnueabihf_)
+* **arm64** (ARM 64-bit, ARMv8 and up: _aarch64-linux-gnu_)
 
 **Supported Ubuntu versions:**
 
-* **Ubuntu 14.04 LTS** (Trusty Tahr)
+* **Ubuntu 14.04 LTS** (Trusty Tahr) - **not available for Node.js 10 and later**
 * **Ubuntu 16.04 LTS** (Xenial Xerus)
+* **Ubuntu 17.10** (Artful Aardvark)
```
Comment (refack) [inline locale/en/download/package-manager.md]: 17.10 went EOL, so it not supported anymore...

**Comment by refack** [comment]
I'm kinda in favor of just linking instead of C&P content (for example next week we're gonna have node v11 as well)...
Maybe just leave a line about supported (as in not EOL RE https://github.com/nodejs/Release/issues/359) node versions on supported distro versions.

**Comment by fhemberger** [comment]
Okay, I removed the whole Debian/Ubuntu/RHEL section and replaced it with a link to the NodeSource repo.

We still need translations/updated for those pages:
- [x] @nodejs/nodejs-es 
- [x] @nodejs/nodejs-ja 
- [ ] @nodejs/nodejs-ko 
- [x] @nodejs/nodejs-uk 
- [x] @nodejs/nodejs-cn 
- [x] @nodejs/nodejs-tw

**Comment by academo** [comment]
I can help with the Spanish translation

File: locale/es/download/package-manager.md [hunk 3]
```diff
@@ -32,189 +31,9 @@ pacman -S nodejs npm
...
-
-**Versiones de Fedora soportadas:**
-
-* **Fedora Rawhide** (i686/x86_64/armv7hl)
-* **Fedora 21** (i686/x86_64/armv7hl)
-* **Fedora 20 (Heisenbug)** (i686/x86_64/armv6hl/armv7hl)
-* **Fedora 19 (Schrödinger's Cat)** (i686/x86_64/armv7hl)
+## Distribuciones de Linux basadas en Debian y Ubuntu, Enterprise Linux/Fedora y Snap
 
+[Official Node.js binary distributions](https://github.com/nodesource/distributions/blob/master/README.md) are provided by NodeSource.
```
Comment (academo) [inline locale/es/download/package-manager.md]:
```suggestion
[Las distribuciones de binarios oficiales de Node.js](https://github.com/nodesource/distributions/blob/master/README.md) son proporcinadas por NodeSource.
```

**Review by academo — CHANGES_REQUESTED** [review]

**Review by refack — APPROVED** [review]

**Comment by fhemberger** [comment]
@academo Thanks!

**Comment by vsemozhetbyt** [comment]
[uk/download/package-manager.md](https://github.com/nodejs/nodejs.org/blob/master/locale/uk/download/package-manager.md) has not translated text, it seems it's just a placeholder. It can be deleted or replaced by the [en/download/package-manager.md](https://github.com/nodejs/nodejs.org/blob/master/locale/en/download/package-manager.md).

**Comment by fhemberger** [comment]
Great, only two more to go.

**Comment by fhemberger** [comment]
I also added missing Android and Solus descriptions to es/ja/ko (untranslated).
Those can be translated in a separate PR as well, as I think it's more important to have the content up to date.

**Comment by fhemberger** [comment]
Thank you all for your help!

**Pull request merged**